<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMailBoxDlg</class>
 <widget class="QDialog" name="AddMailBoxDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加邮箱</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc.qrc">
    <normaloff>:/new/prefix1/image/mailbox.png</normaloff>:/new/prefix1/image/mailbox.png</iconset>
  </property>
  <widget class="QComboBox" name="ui_pop3Port">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>110</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>995</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>账户名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_passwd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>POP3服务器名称：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ui_smtpPort">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>465</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="ui_connectComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>普通</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SSL</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="ui_pop3Server">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>连接安全：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>SMTP服务器名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_user">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_smtpServer">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ui_addButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ui_cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../qrc.qrc"/>
 </resources>
 <connections/>
</ui>
